src/firebase-social-provider.js

```diff
@@ -374,3 +347,180 @@ FirebaseSocialProvider.prototype.setupDetectDisconnect_ = function() {
     }
   }.bind(this));
 };
+
+
+FirebaseSocialProvider.prototype.readFriendProfile_ = function(friendUserId) {
+  return new Promise(function(F, R) {
+    var friendProfileRef = new Firebase(
+        this.allUsersUrl_ + '/' + this.networkName_ +
+        ':' + friendUserId + '/profile/');
+    friendProfileRef.once('value', function(snapshot) {
```

Comment:
Yup, it will invoke the callback each time the entry at the URL is changed.  In practice this means it is invoked for each friend that already exists when you login, and again for each newly added friend after you've signed in.  https://www.firebase.com/docs/web/api/query/on.html